generated/niscope/niscope_service.h

```diff
@@ -125,6 +125,12 @@ class NiScopeService final : public NiScope::Service {
   void Copy(const std::vector<NIComplexNumber_struct>& input, google::protobuf::RepeatedPtrField<niscope_grpc::NIComplexNumber>* output);
   void Copy(const NIComplexI16_struct& input, niscope_grpc::NIComplexInt32* output);
   void Copy(const std::vector<NIComplexI16_struct>& input, google::protobuf::RepeatedPtrField<niscope_grpc::NIComplexInt32>* output);
+  std::map<std::int32_t, std::int64_t> niscopeint64attributevalues_input_map_ { {0, 0},{1, 0},{2, 1}, };
+  std::map<std::int64_t, std::int32_t> niscopeint64attributevalues_output_map_ { {0, 0},{0, 1},{1, 2}, };
+  std::map<std::int32_t, double> niscopereal64attributevalues_input_map_ { {0, 0},{1, 0},{2, 1}, };
+  std::map<double, std::int32_t> niscopereal64attributevalues_output_map_ { {0, 0},{0, 1},{1, 2}, };
+  std::map<std::int32_t, std::string> niscopestringattributevalues_input_map_ {  };
+  std::map<std::string, std::int32_t> niscopestringattributevalues_output_map_ {  };
```

Comment:
We do need to do this to prevent backward compatibility issues in future. We need to generate empty enum and add that in `SetAttribute*` method signature, so that in case new enums are introduced in future - we don't break backward compatibility for the API by updating API signature.

Reply:
The maps are implementation details. If we add them later that wouldn't break compatibility over the wire. The main source of compatibility breakage would be if we change something in the .proto file or reinterpret the meaning or behavior of an attribute value. If it's easier to generate the empty maps to keep the .proto file the same then I can understand that.

Reply:
Ah, that's correct. I could just not add the `generate-mappings` tag to enums if they are empty during metadata mutation in such case. I was thinking it would be difficult to get the information about if enum is empty or not in so many different places where map is referenced, but not adding the tag would do the trick. I will update to remove these maps.

Reply:
Removed the addition of `generate-mappings` completely, since it is not used by any of the drivers yet. Addition of `generate-mappings` will be handled in separate PR later.